src/handler.ts

```diff
@@ -1,17 +1,58 @@
 import 'source-map-support/register';
-import { Context, APIGatewayEvent } from 'aws-lambda';
+import { ScheduledEvent } from 'aws-lambda';
+import { sendModifiedTestStations } from './eventbridge/send';
+import logger from './observability/logger';
+import { DynamoTestStation } from './Interfaces/DynamoTestStation';
 
 const {
   NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE,
 } = process.env;
 
+interface EventDetail {
+  lastModifiedDate: string;
+}
+
+const ONE_DAY = 1;
+
 console.log(
   `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_PROVIDER_STAGE}'\n region: '${AWS_PROVIDER_REGION}'\n\n`,
 );
 
-const handler = (event: APIGatewayEvent, _context: Context) => {
-  console.log('event');
-  console.log(JSON.stringify(event, null, 2));
+const handler = async (event: ScheduledEvent<EventDetail>): Promise<void> => {
```

Comment:
You'll need to test that eventbridge detects an error and implements it's retry policy. I could only get it to work with a callback, which is why I ended up changing it in cvs-tsk-daily-schedules.

Reply:
Well, I had a 50/50 chance - guess I'll change it over to callback then!